api/src/main/resources/workbench.yaml

```diff
@@ -71,6 +71,21 @@ parameters:
     format: int64
     required: true
     description: Data set ID
+  workspaceNamespace:
+    in: path
+    name: workspaceNamespace
+    type: string
+    required: true
+    description: The workspace namespace, aka Google Cloud project ID.
+  workspaceId:
+    in: path
+    name: workspaceId
+    type: string
+    required: true
+    description: >
+      The workspace ID, aka the Firecloud workspace ID. This is actually an immutable string,
+      which in AoU looks like a lower-cased concatenation of the workspace name when it was first
```

Comment:
_looks like_ Is there an actual contract here, or are we in danger of making an assumption about how these are generated that could change on us?